src/tests/class_tests/openms/source/FASTAFile_test.cpp

```diff
@@ -180,15 +216,67 @@ START_SECTION([EXTRA] test_strange_symbols_in_sequence)
   data.push_back(temp_entry);
   data.push_back(temp_entry); // twice
 
+
   file.store(tmp_filename, data);
   file.load(tmp_filename, data2);
-  
+
   ABORT_IF(data2.size() != 2);
   TEST_EQUAL(data2[0] == temp_entry, true);
   TEST_EQUAL(data2[1] == temp_entry, true);
-
 END_SECTION
 
+
+START_SECTION(test_white_spaces)
+//test if spaces and tabulators are removed correctly
+    String tmp_filename;
+    NEW_TMP_FILE(tmp_filename);
+    FASTAFile file;
+    vector<FASTAFile::FASTAEntry> data, data2;
+
+                FASTAFile::FASTAEntry temp_entry;
+                temp_entry.identifier = String("P68509|1433F_BOVIN");
+                temp_entry.description = String("This is the description of the first protein");
+                temp_entry.sequence = String("GDREQLLQRAR LAEQ\tAERYDDMASAMKAVTEL");
+                data.push_back(temp_entry);
+
+                file.store(tmp_filename, data);
+                file.load(tmp_filename, data2);
+
+                ABORT_IF(data2.size() != 1);
+                TEST_EQUAL(data2[0].sequence == string("GDREQLLQRARLAEQAERYDDMASAMKAVTEL"), true);
+
+        END_SECTION
+
+        START_SECTION([EXTRA] test_position) //
+                // test if setPosition() works correctly
+                String tmp_filename;
+                NEW_TMP_FILE(tmp_filename);
+                FASTAFile file;
+                //vector<FASTAFile::FASTAEntry> data;
+                vector<pair<streampos, FASTAFile::FASTAEntry>> data;
+                FASTAFile::FASTAEntry temp_entry;
+                file.readStart(OPENMS_GET_TEST_DATA_PATH("FASTAFile_test.fasta"));
+
+                unsigned i =1;
```

Comment:
ok, the test seems legit.

Can you format the code, so its a bit more readable? Indentation mostly.

And maybe go with simple `for (int i = ..` instead of `while...` ?